<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>445</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <property name="styleSheet" >
   <string notr="true" />
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>27</y>
     <width>613</width>
     <height>418</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="cursor" >
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <widget class="QSplitter" name="splitter" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="childrenCollapsible" >
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="layoutWidget" >
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QListView" name="filterList" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="filterOptions" >
            <property name="frameShape" >
             <enum>QFrame::StyledPanel</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QFrame" name="imageViewFrame" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Sunken</enum>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="thumbView" >
        <property name="maximumSize" >
         <size>
          <width>249</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="alignment" >
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>613</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDebug" >
    <property name="title" >
     <string>Tools</string>
    </property>
    <addaction name="actionDebug" />
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionSaveProject" />
    <addaction name="actionSaveProjectAs" />
   </widget>
   <widget class="QMenu" name="menuProcess" >
    <property name="title" >
     <string>Process</string>
    </property>
    <addaction name="actionStartBatchProcessing" />
    <addaction name="actionStopBatchProcessing" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuProcess" />
   <addaction name="menuDebug" />
  </widget>
  <action name="actionDebug" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Debug</string>
   </property>
  </action>
  <action name="actionSaveProject" >
   <property name="text" >
    <string>Save Project</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveProjectAs" >
   <property name="text" >
    <string>Save Project As ...</string>
   </property>
  </action>
  <action name="actionNextPage" >
   <property name="text" >
    <string>Next Page</string>
   </property>
   <property name="shortcut" >
    <string>PgDown</string>
   </property>
   <property name="autoRepeat" >
    <bool>false</bool>
   </property>
  </action>
  <action name="actionPrevPage" >
   <property name="text" >
    <string>Previous Page</string>
   </property>
   <property name="shortcut" >
    <string>PgUp</string>
   </property>
   <property name="autoRepeat" >
    <bool>false</bool>
   </property>
  </action>
  <action name="actionStartBatchProcessing" >
   <property name="icon" >
    <iconset resource="../resources/resources.qrc" >
     <normaloff>:/icons/16x16/media-playback-start.png</normaloff>:/icons/16x16/media-playback-start.png</iconset>
   </property>
   <property name="text" >
    <string>Start Batch Processing</string>
   </property>
  </action>
  <action name="actionStopBatchProcessing" >
   <property name="icon" >
    <iconset resource="../resources/resources.qrc" >
     <normaloff>:/icons/16x16/media-playback-stop.png</normaloff>:/icons/16x16/media-playback-stop.png</iconset>
   </property>
   <property name="text" >
    <string>Stop Batch Processing</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/resources.qrc" />
 </resources>
 <connections/>
</ui>
